Gemfile.lock

```diff
@@ -3,16 +3,18 @@ GEM
   specs:
     addressable (2.5.1)
```

Comment:
is the the Gemfile.lock file really required in the repo????

Reply:
I just edited it since it was there. It may be a good idea to leave it in, based on https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore
Since this is not a gem,we may want to leave it in the repo.